sum.test.js

```diff
@@ -1,5 +1,5 @@
 const sum = require('./sum');
-
+/*global describe, test, expect */
```

Comment:
this is not the most correct soltuin, read about environments in eslint -> https://eslint.org/docs/user-guide/configuring#specifying-environments

and yes, this was quite simple task